packages/flutter_tools/test/integration.shard/analyze_once_test.dart

```diff
@@ -100,12 +100,27 @@ void main() {
     );
   });
 
-  // testWithoutContext a specific file outside the current directory
-  testWithoutContext('passing one file throws', () async {
+  testWithoutContext('passing one file works', () async {
     await runCommand(
       arguments: <String>['analyze', '--no-pub', libMain.path],
-      exitMessageContains: 'is not a directory',
-      exitCode: 1,
+      statusTextContains: <String>['No issues found!']
+    );
+  });
+
+  testWithoutContext('passing more than one file with errors', () async {
+    final File secondFile = fileSystem.file(fileSystem.path.join(projectPath, 'lib', 'second.dart'))
+      ..createSync(recursive: true)
+      ..writeAsStringSync(r'''import 'package:flutter/material.dart''');
```

Comment:
```suggestion
      ..writeAsStringSync(r"import 'package:flutter/material.dart");
```

Using double quotes makes the syntax error more obvious.